libs/ui-react/.storybook/preview.ts

```diff
@@ -55,7 +55,7 @@ const preview: Preview = {
           ['TextInput', 'Search', 'AddressInput', 'AmountInput'],
           'Communication',
           'Containment',
-          ['Dialog', 'ListItem', 'Tile', 'SheetBar'],
+          ['Dialog', 'ListItem', 'Tile', 'DialogHeader'],
```

Comment:
you can remove it from here, it's an internal component we do no expose it 
```suggestion
          ['Dialog', 'ListItem', 'Tile'],        
```